### ISSUE TITLE
Add cleanup functionality for the canary-release workflow

### ISSUE THREAD

**Issue by jezdez** [conda/actions#30]
We should add an option cleanup feature to the canary-release workflow that allows deleting the channel's N packages (or by time) automatically when running the workflow. That way we reduce the risk of overflowing the hard disk limits of anaconda.org account (10G).

**Comment by kenodegard** [comment]
Is there an API call for this?

cc: @chenghlee

**Comment by chenghlee** [comment]
Yeah; maybe `DELETE /release/{owner}/{package}/{version}`? See https://api.anaconda.org/docs.

**Comment by kalawac** [comment]
Do we still want this? It's about to be closed as stale.
